flow/e2e/test_utils.go

```diff
@@ -540,11 +540,25 @@ func (l *TStructuredLogger) Error(msg string, keyvals ...interface{}) {
 }
 
 func CompareTableSchemas(x *protos.TableSchema, y *protos.TableSchema) bool {
```

Comment:
eventually want to compare modifiers, but probably not time for that yet until after numeric fixes